### PR TITLE
Updated the location of the "Next" button on the Submission Guide page

### PR DESCRIPTION
**Pull request by dfrasier**
Removed the right rail and moved the “Next button” to the bottom of the
page.

### CODE REVIEW

File: src/js/helpers/generateFilesHelper.js [hunk 1]
```diff
@@ -39,9 +39,7 @@ export const generateFile = (type, submissionId, start, end) => {
             .end((errFile, res) => {
 
                 if (errFile) {
-                    const response = Object.assign({}, res.body);
-                    response.httpStatus = res.status;
-                    deferred.reject(response);
```
Comment (kevinli-work) [inline src/js/helpers/generateFilesHelper.js]: Why were these lines deleted?

**Comment by kevinli-work** [comment]
Please make sure you're working off the latest `dev`, this PR recommits deleted files from `/src/_sccss2`

File: src/js/helpers/generateFilesHelper.js (hunk 1 above)
Comment (dfrasier) [inline src/js/helpers/generateFilesHelper.js]: I'm not sure why these lines are deleted. I didn't even open this file.

**Comment by kevinli-work** [comment]
Pushed additional merges to resolve the `generateFilesHelper` incorrect merge and updated to the latest `dev`. @mab6bf should review
